scss/_patterns_code-copyable.scss

```diff
@@ -3,59 +3,49 @@
 $cc-button-size: 36px;
 
 @mixin vf-p-code-copyable {
-  .p-code-copyable {
-    @extend %bordered-text-vertical-padding;
-    @extend %vf-bg--x-light;
-    @extend %vf-has-round-corners;
-    @extend %vf-is-bordered;
+  @include deprecate('3.0.0', 'p-code-copyable is being removed, use the code snippet block instead.') {
+    .p-code-copyable {
+      background-color: $color-code-background;
+      display: flex;
```

Comment:
Is display flex still needed if button is hidden?

Reply:
No, good spot.